csharp/ql/test/library-tests/assignables/Assignables.cs

```diff
@@ -132,4 +132,11 @@ void DelegateRef(Delegate d)
         var x = 0;
         d(ref x, out string s);
     }
+
+    void UsingDeclarations()
+    {
+        using var x = new System.IO.MemoryStream();
+    }
 }
+
+// semmle-extractor-options: /langversion:8.0
```

Comment:
It seems easier to leave it as-is and is consistent with the way it's done elsewhere. It might be worth reorganising the `semmle-extractor-options` at a later date once we decide what the consistent approach should be.